<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowFilesDialog</class>
 <widget class="QDialog" name="ShowFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
  <action name="actionInsert_a_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/insert.png</normaloff>:/insert.png</iconset>
   </property>
   <property name="text">
    <string>Insert a file</string>
   </property>
  </action>
  <action name="actionExtract_this_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/extract.png</normaloff>:/extract.png</iconset>
   </property>
   <property name="text">
    <string>Extract this file</string>
   </property>
  </action>
  <action name="actionDelete_this_file">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/deleteThisFile.png</normaloff>:/deleteThisFile.png</iconset>
   </property>
   <property name="text">
    <string>Delete this file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
